todo/settings.py

```diff
@@ -11,7 +11,7 @@
 """
 
 import os
-import django_heroku
```

Comment:
I miss `django_heroku` 😞.

Reply:
😢 had to do so much of new config just to makeup for heroku

Reply:
> I miss django_heroku 😞.

+1 🥲